app/views/stages/_fields.html.erb

```diff
@@ -61,6 +61,14 @@
     </div>
   </div>
 
+  <div class="form-group">
+    <%= form.label :name, "Template Environment", class: "col-lg-2 control-label" %>
+    <div class="col-lg-4">
+      <%= form.select :template_environment_id, Environment.all.collect {|e| [e.name, e.id] }, { include_blank: true }, {class: 'form-control' } %>
```

Comment:
How do we mark which stages are templates for which environment, then? Maybe `stage.deploy_groups.environment`? I don't know. That would let them choose more than one template per env, and we'd be, like, "which one do we use?"

This makes it explicit. Someone has to look at the stages and say "This stage, this environment". And nothing can go wrong.

I appreciate a normalized database as much as the next guy, but people aren't normalized.